src/backend/distributed/commands/table.c

```diff
@@ -4130,10 +4130,7 @@ ConvertNewTableIfNecessary(Node *createStmt)
 
 		if (ShouldCreateTenantSchemaTable(createdRelationId))
 		{
-			ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-							errmsg("cannot create a tenant table using "
-								   "CREATE TABLE AS or SELECT INTO "
-								   "statements")));
+			CreateTenantSchemaTable(createdRelationId);
 		}
```

Comment:
should we be checking for below as well?

```c
	/* not try to convert the table if it already exists and IF NOT EXISTS syntax is used */
	if (baseCreateTableStmt->if_not_exists && IsCitusTable(createdRelationId))
	{
		return;
	}
```

and in regression tests, let's also try to create the same table by using that syntax

Reply:
(Ideally we should be able to make use of CreateStmt code for CreateTableAsStmt as well --because it respects if_not_exist--, but doing so would result in supporting CreateTableAs commands for Citus managed tables too, which is not in the scope of this PR.)